pkg/gofr/datasource/cassandra/mock_interfaces.go

```diff
@@ -1,3 +1,6 @@
+//go:build exclude
```

Comment:
I just have one question, will this change get reverted when someone generates a new mock using mockgen - then this change has to be kept in mind.

Reply:
Yes, these changes will be reverted when a new mock is generated, as they are manually added. Once mockgen is run, it will overwrite these modifications automatically. To address this, I considered creating an internal directory to hold all test and mock files, which would be excluded from generating the GoDocs. However, this led to circular dependency issues, requiring changes across the entire codebase. That's why I opted for the current approach of manually adding the modifications.

Reply:
Other contributors would have to make sure that these are included whenever there is any change in the mock files.